scripts/skills/perks/perk_rf_phalanx.nut

```diff
@@ -46,6 +54,30 @@ this.perk_rf_phalanx <- ::inherit("scripts/skills/skill", {
 		}
 	}
 
+	function onTurnStart()
+	{
+		local actor = this.getContainer().getActor();
+		local myTile = actor.getTile();
+		for (local i = 0; i < 6; i++)
```

Comment:
This is less about your PR:
I would really like to have a `function getAdjacentActors()` for the `actor.nut` class which returns an array of up to 6 actor references.

MSU currently only offers a `getAlliedActors` but that is not efficient when we know that we only need to check the adjacent tiles.

Reply:
You can propose this idea as an issue over on the MSU repo. We will discuss it. But I want to point out that currently this can be done using the MSU-added `::Tactical.Entities.getActorsByFunction` as:
```squirrel
local myTile = actor.getTile();
local actors = ::Tactical.Entities.getActorsByFunction(@(a) a.isPlacedOnMap() && a.getTile().getDistanceTo(myTile) == 1);
```
This will give all actors even if they are allied or not. If we only want allied actors then we can just add `a.isAlliedWith(actor)` condition in there.